bin/supply

```diff
@@ -57,18 +59,16 @@ chmod +x "${BUILD_DIR}/.profile.d/00-test-endpoint.sh"
 chmod +x "${BUILD_DIR}/.profile.d/02-redirect-logs.sh"
 chmod +x "${BUILD_DIR}/.profile.d/01-run-datadog.sh"
 
-# export DD environment variables
+# import helper functions
 . "${DATADOG_DIR}/scripts/utils.sh"
+
+# export DD environment variables
 dd_export_env "${DATADOG_DIR}/.raw_datadog_env"
 
 # sanitize env vars and export a new a env file
 python "${DATADOG_DIR}/scripts/parse_env_vars.py" "${DATADOG_DIR}/.raw_datadog_env" "${DATADOG_DIR}/.datadog_env"
 
-# export DD_TAGS for ddtrace
-DD_TAGS=$(python "${DATADOG_DIR}"/scripts/get_tags.py)
-export DD_TAGS
-DD_DOGSTATSD_TAGS=$(python "${DATADOG_DIR}"/scripts/get_tags.py)
-export DD_DOGSTATSD_TAGS
+export DD_TAGS=$(python "${DATADOG_DIR}/scripts/get_tags.py")
 
 # mark the script as finished, useful to sync the update_agent_config script
```

Comment:
I think supply and compile should have the same content

Reply:
Done!